app/assets/javascripts/miq_application.js

```diff
@@ -320,7 +320,10 @@ function miqCheckForChanges() {
     }
   }
   // use default browser reaction for onclick
-  return true;
+  if(obj && (href = obj.getAttribute("href")))
```

Comment:
@himdel Thanks! That does seem to catch the urls that are not previously defined by ```$stateProvider.state```.
Also this is a much better approach compared to using ```ui-sref``` for every link.